weave-js/src/components/PagePanelComponents/Home/Browse3/filters/common.ts

```diff
@@ -247,7 +249,7 @@ export const getOperatorValueType = (operatorValue: string): string => {
 
 export const getOperatorOptions = (field: string): SelectOperatorOption[] => {
   const fieldType = getFieldType(field);
-  if ('id' === fieldType) {
+  if (STRING_LIKE === fieldType) {
```

Comment:
Something isn't right here - this doesn't have `contains`

Reply:
I think maybe we want to keep `id` as its own special fieldType - I'm not sure e.g. "does not contain" on an id is useful.